src/main/java/entities/internalUserList.java

```diff
@@ -0,0 +1,8 @@
+package entities;
+
+import java.util.ArrayList;
+
+public class internalUserList {
+
```

Comment:
not 100% sure about this but wouldn't you need a getter/setter to access and modify this list?

Reply:
Yea I think so--see User Story